src/desktop/apps/home/routes.coffee

```diff
@@ -45,21 +45,24 @@ fetchMetaphysicsData = (req, showHeroUnits)->
       "query-input": "required name=search_term_string"
     }
   }
-
+  
+  showCollectionsHubs = res.locals.sd.COLLECTION_HUBS == "experiment"
   res.locals.sd.PAGE_TYPE = 'home'
   initialFetch = Q
     .allSettled [
-      fetchMetaphysicsData req, true
+      fetchMetaphysicsData req, true, showCollectionsHubs
       featuredLinks.fetch cache: true
     ]
   initialFetch
     .then (results) ->
       homePage = results?[0].value.home_page
+      collectionsHubs = results?[0].value.marketingCollections
       heroUnits = homePage.hero_units
       heroUnits[positionWelcomeHeroMethod(req, res)](welcomeHero) unless req.user?
 
       res.locals.sd.HERO_UNITS = heroUnits
       res.locals.sd.USER_HOME_PAGE = homePage.artwork_modules
+      res.locals.sd.COLLECTIONS_HUBS_DATA = collectionsHubs
```

Comment:
I was a little surprised to find that, in order to pass the hubs to the .jade template, I had to pass them via sharify instead of passing them in via the `res.render` call down on [line 71](https://github.com/artsy/force/pull/4476/files#diff-e7898546a64e7081f4c581fae8fc6a35R71). But that seemed to be the case - adding them after [line 77](https://github.com/artsy/force/pull/4476/files#diff-e7898546a64e7081f4c581fae8fc6a35R77) did _not_ give me access to them in the template, while passing them via sharify here did. 

(Saying that out loud makes me feel like I missed something simple, so if you feel confident that I _should_ have been able to pass them in that way, please speak up.)

Reply:
Yeah this didn't sound quite right to me either, so I faffed a little and you can see the results here: 

https://github.com/anandaroop/force/commit/9c529fbd5cafb199a53caafd35c8f4a3fa35a92e

This did work for me (after I remembered to change the [conditional on L40](https://github.com/anandaroop/force/commit/9c529fbd5cafb199a53caafd35c8f4a3fa35a92e#diff-0d6a4eed0ca05fe07465620d70b329ccR40))

Reply:
It worked! My guess is that I referenced something named `collectionsHubs` in one place, and `collectionHubs` in another. This change has been made & force-pushed.